client/app/src/main/java/io/mosip/registration/app/activites/JobServiceActivity.java

```diff
@@ -129,27 +125,33 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
                 if (isChecked) {
                     try {
                         int resultCode = jobServiceHelper.scheduleJob(jobServiceModel.getId(), jobServiceModel.getApiName());
-
                         if (resultCode == JobScheduler.RESULT_SUCCESS) {
                             Log.d(TAG, "Job scheduled");
+                            jobServiceModel.setEnabled(true);
+                            jobServiceModel.setLastSyncTime(jobServiceHelper.getLastSyncTime(jobServiceModel.getId()));
                             Toast.makeText(JobServiceActivity.this, "Job scheduled", Toast.LENGTH_SHORT).show();
                         } else {
                             Log.d(TAG, "Job scheduling failed");
                             Toast.makeText(JobServiceActivity.this, "Job scheduling failed", Toast.LENGTH_SHORT).show();
+                            jobServiceModel.setEnabled(false);
+                            compoundButton.setChecked(false);
                         }
                     } catch (ClassNotFoundException e) {
                         Log.e(TAG, "Job scheduling failed : service " + jobServiceModel.getApiName() + " not implemented", e);
                         Toast.makeText(JobServiceActivity.this, "Job scheduling failed : service " + jobServiceModel.getApiName() + " not implemented", Toast.LENGTH_SHORT).show();
+                        jobServiceModel.setEnabled(false);
```

Comment:
Don't add any logic in the catch block. The else block and the logic in the catch block are the same. This need 
 restructure.